Tests/ClientTests/Frontend/Browser/TabManagement/Tab+ChangeUserAgentTests.swift

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+@testable import Client
+
+import XCTest
```

Comment:
nit, only one import XCTest necessary
```suggestion
import XCTest

@testable import Client
```